cmd/version/githubclient.go

```diff
@@ -0,0 +1,219 @@
+package version
```

Comment:
Why not use an existing library like https://github.com/google/go-github?
Feels like we're reinventing stuff and have to then maintain it.

Reply:
go-github has lot more than what we need. So I just took that one part without importing another library.